dev/Microsoft.Process.Environment/EnvironmentManager.cpp

```diff
@@ -63,4 +74,125 @@ namespace winrt::Microsoft::ProjectReunion::implementation
     {
         throw hresult_not_implemented();
     }
+
+    StringMap EnvironmentManager::GetProcessEnvironmentVariables()
+    {
+        //Get the pointer to the process block
+        LPWSTR environmentVariablesString = GetEnvironmentStrings();
```

Comment:
I believe it's preferable to use PWSTR instead of LPWSTR.  They are the same but LPWSTR is obsolete.

Reply:
Yes. Avoid L-prefixes. They're from days of yore when Win16 and near+far pointers roamed the earth... ;-)

Very passe'. Current coding standards say not to use them anymore

Reply:
Thank you for the history lesson.  Didn't know those were a thing.  The L prefix has been removed and I'll work on getting all the L's removed.